components/raftstore/src/store/worker/split_controller.rs

```diff
@@ -840,18 +840,28 @@ impl AutoSplitController {
             if recorder.is_ready() {
                 let key = recorder.collect(&self.cfg);
                 if !key.is_empty() {
+                    info!(
+                        "load base split region";
+                        "region_id" => region_id,
+                        "qps" => qps,
+                        "byte" => byte,
+                        "cpu_usage" => cpu_usage,
+                        "split_key" => log_wrappers::Value::key(&key),
+                    );
                     split_infos.push(SplitInfo::with_split_key(
                         region_id,
                         recorder.peer.clone(),
                         key,
                     ));
                     LOAD_BASE_SPLIT_EVENT.ready_to_split.inc();
-                    info!("load base split region";
-                        "region_id" => region_id,
-                        "qps" => qps,
-                        "byte" => byte,
-                        "cpu_usage" => cpu_usage,
-                    );
+                    if recorder.hottest_key_range.is_some() {
+                        // debug message
+                        info!(
+                            "DEBUG : load based split : expensive range";
+                            "hot start_key" => log_wrappers::Value::key(&recorder.hottest_key_range.as_ref().unwrap().start_key),
+                            "hot end_key" => log_wrappers::Value::key(&recorder.hottest_key_range.as_ref().unwrap().end_key),
+                        );
```

Comment:
Printing the hottest key range here feels strange because, when the collected key is not empty, the CPU-based split strategy does not apply in this branch. Line 867 seems like a better place to display this information.

Reply:
we are already printing the hot key in line 891. Printing the hot key here will give us additional information that if split key is in the hot range.

Reply:
are you ok with this change ?

Reply:
This change is okay for me, but the code can be simplified a bit. These two new fields can be directly added to the original info at line 849.